### ISSUE TITLE
Rebrand Angular 2/4/5?

### ISSUE THREAD

**Issue by Coruscate5** [angular/angular#19318]
## I'm submitting a...
[ X ] Feature request

This is more of a general point - as I've been grappling with the framework for a major production application, I've come across posts/SO questions/blogs that reference the "new" Angular in all sorts of different ways. 

I can read half a post without realizing that they are in fact referencing AngularJS/1.X - and date stamps don't necessarily hold meaning/help too much either (since many are still developing in AngularJS/1.X). 

An internal coworker was presenting the "new" Angular for quite a while using the phrase "**Angular 2**" which has been replaced nearly twice-over (by 4 & 5, with a trend that will likely continue with major version releases).

Can we give it a more identifiable name? Something that has to do with its distinct architecture that will likely _not_ change on a regular basis - e.g. _AngularTS_? 

Other (just for fun) Suggestions: 

- AngularX
- Angular++
- Angular-MinTS

This would make searchability, branding, & likely adoption quite a bit easier on the developer masses.. Microsoft pulled a similar move with .NET Core (originally just called ASP.NET 5), which helped me/others enormously. 

Apologies if this has already been suggested & I missed it - but I've run into this so many times I figured I'd get it out there in case it wasn't already.

**Comment by Toub** [comment]
You can use "angular -angularjs" in your search engine to exclude angularJS/1.x results.

**Comment by Coruscate5** [comment]
Thanks @Toub - I'm aware of several "tricks" I can use to narrow/hone-in on my search results. I think the point I'm making here is that things would be easier if there was a distinct name - provided someone on the team simply just considers the merits/cons of the suggestion, that's good enough for me..

**Comment by ericmartinezr** [comment]
https://github.com/angular/angular/issues/14802

**Comment by Coruscate5** [comment]
@ericmartinezr - hmm, this issue seems to have swirled more around the Search issue rather than the overarching branding. I'm a bit surprised by the negativity around the AngularTS suggestion (I'm guessing it's because they are worried renaming again won't help, though I feel like there are valid points that make this angular markedly different - Again, .NET Core went through several branding iterations & I think settled in a good place).

Either way, take React-Native as a counterexample - one could make a specious argument based on trends that React-Native as a framework is "more popular" based on the confusion of trying to isolate the share of Angular (the term itself is a word, plus the version difference). Honestly, I couldn't really get a handle on the market share analysis after a bit of tweaking.

**Comment by Coruscate5** [comment]
Just for reference, I did just read the [branding blog article ](http://angularjs.blogspot.com/2017/01/branding-guidelines-for-angular-and.html) - I don't disagree with any of the principles laid out there, but knowing what I know now I would still consider changing the "primary" moniker, based on some of the issues I've laid out. I fully expect the answer here to be just a flat "no", but it's never too late to change your mind.

**Comment by IAMtheIAM** [comment]
I agree that it is really bad to have one product named AngularJS and the other named Angular... people constantly mix them up, plus search engines don't know the difference. Its a time waster to ALWAYS have to type "something I want for Angular -angularjs". 

AngularJS should be rebranded to something comlpetely different like "an old js library" or "no longer needed". because it is obsolete now that Angular 2+ is out.  

Branding took a major hit with this mixup.

**Comment by DeveloperAlex** [comment]
Or maybe "ngx".

**Comment by IAMtheIAM** [comment]
@DeveloperAlex you mean for angular or angularjs to be called NGX?

**Comment by IgorMinar** [comment]
We've spent a lot of time discussing our branding in the past. This resulted in the branding guidelines that we published a year ago http://angularjs.blogspot.com/2017/01/branding-guidelines-for-angular-and.html and https://angular.io/presskit#brand-names.

We expected that it will take some time for the guidelines to get widely adopted, but in reality most of the highly visible resources have been updated to use the branding correctly long time ago. Since then we've seen good adoption of the guidelines and we currently have no intention to change them.

If you see someone using the terminology incorrectly, you can help by pointing them to the blog post I mentioned above. Thank you!

**Comment by Coruscate5** [comment]
@IgorMinar Thanks, this was the answer I expected - all I'm asking is to keep an open mind, since as a relatively new developer to the framework I've still found the Angular world rife with confusion (aside from the SEO difficulties) 8 months after the blog post was written.

**Comment by mbejda** [comment]
**Rename Angular to Angular Typescript!**

**Comment by KErez** [comment]
Actually it is not to late to rebrand Angular.
Even on this thread you see example of famous successful rebranding of .Net.
Also Angular was called at the beginning angular 2 and only after a while was decided to leave the number from the name.
It is understandable that there was a committee but that part was misssed.
Also rebranding angularjs in retrospect will not magically change the enormous amount of articles that already exist with the name "angular"
I believe that rebranding it on the next version to AngularX/AngularTS will be praised by the community
